packages/aragon-wrapper/src/rpc/handlers/search-identities.js

```diff
@@ -0,0 +1,11 @@
+export default function (request, proxy, wrapper) {
+  const [searchTerm] = request.params
+
+  if (searchTerm.length < 3) {
+    return Promise.reject(
```

Comment:
Why `reject` over answering with an empty array?

Reply:
After thinking, for no good reason.